src/main/resources/archetype-resources/frontend/src/main/resources/index.html

```diff
@@ -0,0 +1,11 @@
+<html lang="en">
```

Comment:
We don't need this anymore, right?
We deliver only JS, the HTML for embedding it will always be the atlassian plugin

Reply:
That's right, removed it.